docs/content/develop/resource-reference.md

```diff
@@ -26,7 +26,9 @@ Example:
 ```yaml
 description: |
   This is a multi-line description
-  of a resource.
+  of a resource. All multi-line descriptions must follow
```

Comment:
Is this a new limitation of the Go-format YAMLs? I guess the other way to make multi-line descriptions would be to add newlines into the text?

Reply:
I was under the impression we had some description diffs that were impossible to remove until the yaml conversion template was modified to use this approach, but I suppose it's possible that `\n`s in a standard quotation wrapped string would still work? I would still prefer to recommend this approach as it would be easier to read.

Reply:
Ah I see. Yes we wanted to use the | format to just reduce the number of diffs generated. Without using it, the generator was producing a unreviewable amount of diffs.

So I guess it's a recommendation rather than a necessity.

Reply:
I'll replace it with a "should" to lower the intensity